adm_program/system/bootstrap.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+ ***********************************************************************************************
+ * Bootstrap non DB things
+ *
+ * @copyright 2004-2017 The Admidio Team
+ * @see https://www.admidio.org/
+ * @license https://www.gnu.org/licenses/gpl-2.0.html GNU General Public License v2.0 only
+ ***********************************************************************************************
+ */
+
+// embed config and constants file
+$rootPath = substr(__FILE__, 0, strpos(__FILE__, DIRECTORY_SEPARATOR . 'adm_program'));
+require_once($rootPath . '/adm_program/system/init_globals.php');
+require_once($rootPath . '/adm_program/system/constants.php');
+
+// ERROR REPORTING
+// http://www.phptherightway.com/#error_reporting
+// https://secure.php.net/manual/en/errorfunc.configuration.php
+// error_reporting(E_ALL | E_STRICT); // PHP 5.3 fallback (https://secure.php.net/manual/en/function.error-reporting.php)
+ini_set('error_reporting', '-1');
+ini_set('log_errors', '1');
+
+if ($gDebug)
+{
+    ini_set('display_errors', '1');
+    ini_set('display_startup_errors', '1');
+}
+else
+{
+    ini_set('display_errors', '0');
```

Comment:
Ok. Idea:
`$gDebug` is not set -> use default
`$gDebug` is true -> set values to `'1'`
`$gDebug` is false -> set values to `'0'`